dashboard/db/migrate/20161019233302_create_schools.rb

```diff
@@ -0,0 +1,11 @@
+class CreateSchools < ActiveRecord::Migration[5.0]
+  def change
+    create_table :schools, :id => false do |t|
+      t.integer :id, limit: 8, comment: 'NCES public school ID'
+      t.references :school_district, foreign_key: true
+      t.string :name
+      t.string :school_type
+      t.timestamps
+    end
+  end
+end
```

Comment:
Any the chance the migration can be merged before the API routes accessing the table are merged?